Scripts/Prerequisite.ps1

```diff
@@ -0,0 +1,96 @@
+param(
+    [Parameter(Mandatory = $true)]
+    [string]$subscriptionId,
+    [Parameter(Mandatory = $true)]
+    [string]$resourceGroupName,
+    [Parameter(Mandatory = $true)]
+    [string]$keyVaultName,
+    [Parameter(Mandatory = $true)]
+    [string]$location,
+    [Parameter(Mandatory = $true)]
+    [string]$certName,
+    [Parameter(Mandatory = $true)]
+    [string]$keyVaultSecretNameForServicePrincipal,
```

Comment:
Not clear from name what is this? Can it be simplified to 'SecretKey'

Reply:
@Shuvajyoti  - Can i have name as ServicePrincipalSecret?